haiku-apps/bshisen/bshisen-1.2.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="BeOS tile matching game"
+DESCRIPTION="BShisen is a Majong style game made for Haiku."
+HOMEPAGE="https://github.com/HaikuArchives/BShisen"
+COPYRIGHT="1998-1999 Kelvin W Sherlock"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="1e86e4daec9f48d9aa9642227f962159cfe3a733"
+SOURCE_URI="https://github.com/HaikuArchives/BShisen/archive/$srcGitRev.tar.gz"
+#CHECKSUM_SHA256="e0b9844cebb6ed2cf42f93e592fe2e4ce0b801457b31cbac7ff7f5b5d493f76e"
```

Comment:
```diff
-#CHECKSUM_SHA256="e0b9844cebb6ed2cf42f93e592fe2e4ce0b801457b31cbac7ff7f5b5d493f76e"
+CHECKSUM_SHA256="263343a257a313358fee3b8bf3448cb50ab15f8e4c8b0ecd7e12b2e408822ba6"
```

BTW, the game is working perfectly on x86_gcc2 :)

Reply:
I intended to wait until after school, but thanks for testing it for me @fbrosson :)